doc/osh.1

```diff
@@ -5,45 +5,47 @@
 .Nm osh
 .Nd A bash-compatible UNIX shell
 .Sh SYNOPSIS
-.Nm 
+.Nm
 .Op Ar options
 .Ar script
 .Op Ar args
-.Br
-.Nm 
+.Nm
 .Op Ar options
 .Fl c
 .Op command_string
 .Op args
 .Sh DESCRIPTION
-OSH is a command-line shell. It behaves like busybox and `osh` is a symlink to it.
+.Nm OSH
+is a command-line shell.
+It behaves like busybox and `osh` is a symlink to it.
 .Sh OPTIONS
 OSH accepts POSIX \fBsh\fR flags, with the following differences:
```

Comment:
```mdoc
.Nm
accepts POSIX
.Xr sh 1
flags, with the following differences:
```

This is good for a first start, but I recommend in the future listing all of these flags.